### DISCUSSION TITLE
Alternative that will disable extensions, you must be able to change your network settings

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#226] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
https://www.youtube.com/watch?v=SMUwz6K0KnY&t=0s

**Comment by skoopy-star** [comment]
i dont have goguardian does it matter

**Reply by The-Man-Is-Me** [reply]
try it but it may not work

**Comment by 1-Mythical** [comment]
I don't have GoGuardian. They need to make it work with either Blocksi or Lightspeed Filter Agent because just GoGuardian isnt working

**Comment by CheezeKake77** [comment]
Is the IP down? I think kozmik isn't hosting it at the moment

**Reply by bartholemuew** [reply]
this was in nov

**Comment by Aka-but-nice** [comment]
@yeeteeyt yo who made this, how did they make this, can they make one for securly?

**Reply by MilkMaker23** [reply]
can they make one for lightspped
